Implemented feedback screen skeleton (#130)

arandr opened the pull request.

intrications commented:
I think the send icon should be 24dp not 36dp:

app/src/main/res/drawable-xhdpi/ic_send_white_36dp.png

also please add all the sizes (mdpi/hdpi/xhdpi/xxhdpi/xxxhdpi)

Are you getting the icons from here: https://github.com/google/material-design-icons/blob/master/content/drawable-mdpi/ic_send_white_24dp.png

intrications commented on app/src/main/res/layout/activity_feedback.xml:
```diff
@@ -0,0 +1,133 @@
+        <EditText
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@string/feedback_default_email"
```
android:text should be android:hint so that the text doesn't have to be deleted when the user clicks on it

intrications commented on app/src/main/res/layout/spinner_dropdown_feedback.xml:
```diff
@@ -0,0 +1,11 @@
+    android:id="@android:id/text1"
+    style="?android:attr/spinnerDropDownItemStyle"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
```
Try adding android:minHeight="?android:attr/listPreferredItemHeightSmall"

The height is too small to easily click at the moment.

intrications commented on app/src/main/res/values/strings.xml:
```diff
@@ -62,5 +62,25 @@
     <string name="about_Pp">Privacy Policy</string>
     <string name="about_Oss">Used Open Source Libraries</string>
     <string name="title_activity_os_lib">About</string>
+    <string name="title_activity_feedback">FeedbackActivity</string>
```
Remove "title_activity_feedback". It will never be used in the app.

intrications commented on app/src/main/res/values/strings.xml:
```diff
@@ -62,5 +62,25 @@
+
+    <string-array name="feedback_topics">
+        <item>Suggest new features</item>
+        <item>Report issues with the iOS app</item>
```
Remove "Report issues with the iOS app". See my comment on https://assembly.com/kanshu/bounties/130

intrications commented:
Delete this: app/src/main/res/drawable-mdpi/feedback_header_bg.9.png

And create the tabs using the same method as the other screens. Ignore the mockup - make it look like Android.

intrications commented:
The left edge of the EditTexts should be vertically aligned. You may need to use something like <Table> or <GridView>

![device-2015-02-08-092926](https://cloud.githubusercontent.com/assets/1798519/6096080/5791b4fa-af77-11e4-9a69-f934b435b653.png)

fernandobuzi commented:
@arandr Question: will the user be able to tell that the topic area is a drop-down menu?

alouanemed commented:
@fernandobuzi  Nope ! that's and editText view

arandr commented:
@intrications  About the header being a tab, it looked to me as if it could be one, but I was confused as to why we needed a tab view with a single tab in it? Is there going to be an extra tab later? Otherwise why have a header/tab at all?

arandr commented:
So I fixed all of that except the tab thing (still not understanding why we need tabs, and it doesn't seem right to use something that implies functionality if it's just for look-and-feel). I'll do the tabbed view tonight anyway if I have time since you guys are all in different timezones.
@alouanemed  @fernandobuzi  The dropdown does have an arrow, but it was not clearly visible (because of its color in the theme). Changed that so that the arrow is more visible.

intrications commented:
@arandr I agree there should be no tab and "Kanshu" in the ActionBar should be "Send Feedback". @fernandobuzi What do you think? I know the mockups have the tab but it doesn't really follow Android design guidelines to have one tab on it's own. It is just taking up space.

fernandobuzi commented:
@intrications @arandr I agree that we remove both. What we need is to somehow make sure that the screens we did before follow the same guidelines. But we can polish that up later, like you suggested @intrications.
